include/dmlc/blockingconcurrentqueue.h

```diff
@@ -5,6 +5,9 @@
 // Uses Jeff Preshing's semaphore implementation (under the terms of its
 // separate zlib license, embedded below).
 
+#ifndef DMLC_BLOCKINGCONCURRENTQUEUE_H_
+#define DMLC_BLOCKINGCONCURRENTQUEUE_H_
+
 #pragma once
```

Comment:
This file just went through reformatter. No changed of consequence other than dmlc namespace wrapping and  DMLC_BLOCKINGCONCURRENTQUEUE_H_ definition.